ios/RNCallKeep/RNCallKeep.m

```diff
@@ -304,6 +304,20 @@ - (void)requestTransaction:(CXTransaction *)transaction
     }];
 }
 
++ (BOOL)checkIfActiveCall:(NSString *)uuidString
+{
+    CXCallObserver *callObserver = [[CXCallObserver alloc] init];
+    NSUUID *uuid = [[NSUUID alloc] initWithUUIDString:uuidString];
+
+    for(CXCall *call in callObserver.calls){
+        NSLog(@"Connected: %d", [call.UUID isEqual:uuid]);
```

Comment:
Can you change the log to `NSLog(@"[RNCallKeep] isCallActive %@ %d ?", call.UUID, [call.UUID isEqual:uuid]);` ?